test/Interop/Cxx/enum/anonymous-with-swift-name-objc-module-interface.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=AnonymousWithSwiftName -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
+//
+// REQUIRES: objc_interop
+
+// CHECK: class ColorMaker {
+// CHECK:   class func makeColor(withOptions opts: SOColorMask)
+// CHECK:   func makeColor(withOptions opts: SOColorMask)
```

Comment:
I suspect you will have to update these now that https://github.com/apple/swift/pull/42412 is merged.